pkg/ovsdb/database.go

```diff
@@ -12,28 +13,28 @@ import (
 	clientv3 "go.etcd.io/etcd/client/v3"
 	"go.etcd.io/etcd/client/v3/concurrency"
 	"k8s.io/klog/v2"
-
-	"github.com/ibm/ovsdb-etcd/pkg/ovsjson"
 )
 
 type Databaser interface {
 	GetLock(ctx context.Context, id string) (Locker, error)
+	AddMonitors(dbName string, updaters map[string][]*updater, handler handlerKey)
+	RemoveMonitors(dbName string, updaters map[string][]string, handler handlerKey)
+    RemoveMonitor(dbName string)
```

Comment:
following my comment (that we should not use the database abstraction) it would be better to call clientv3 directly from monitor implementation